examples/gcs.rs

```diff
@@ -1,4 +1,4 @@
-use log::info;
+use log::*;
```

Comment:
Why the star import?

Reply:
Because we actually import everything exported from the same module here; it seems more readable than providing a list of imports here.